src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -205,38 +205,32 @@ private void createToolBar()
         // Events
         toolitemConnect.addSelectionListener(new SelectionListener()
         {
-            @Override
             public void widgetDefaultSelected(SelectionEvent e)
             {
             }
 
-            @Override
             public void widgetSelected(SelectionEvent e)
             {
                 defaultActionRequested(getSelectedTreeItem());
             }
         });
         toolitemPortForwarding.addSelectionListener(new SelectionListener()
         {
-            @Override
             public void widgetDefaultSelected(SelectionEvent e)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)